### ISSUE TITLE
Add sim object locations in Scene Summary for initial orientation only

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#393]
As discussed yesterday (April 10) with @emily-phet and @jessegreenberg , for initial orientation we need to make sure users create a mental model that the _Yellow balloon_ starts off in between the Sweater and the Wall. This was clear before I removed location details from the Scene Summary. I still think the simplified Scene Summary without location is a good design choice, but for page load and reset all we need to add in an **orientation phrase**:

  - "Yellow balloon is at center of Play Area, evenly between sweater and wall. Sweater is at far left, wall at far right.”

This **orientation phrase** can be placed in a bullet right after the three objects are mentioned:
- Currently, room has {{a yellow balloon}}, {{a sweater}} and {{a removable wall}}. 
- _{{Yellow balloon is at center of Play Area, evenly between sweater and wall. Sweater is at far left, wall at far right.}}_
- [Balloon charge information]
- [Sweater and Wall charge information]

**Note** the orientation phrase could alternatively be appended to the first bullet, if that is easier.

This sim is in Dev Testing, so tagging phetsims/QA#112.

**Comment by terracoda** [comment]
@jessegreenberg, let me know if you need any other information.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, Ill add this.

> Note the orientation phrase could alternatively be appended to the first bullet, if that is easier.

It would be easier to create a new bullet in the list.

The phrase should disappear when an object is moved or removed from the play area.

**Comment by terracoda** [comment]
Yes, that logic would be perfect! 

And that would mean if the user just changed one of the Charge Settings `radiobuttons` and then read the Scene Summary again, the orientation phrase would still be there, right?

**Comment by jessegreenberg** [comment]
@terracoda great! Yes, that is correct.

**Comment by jessegreenberg** [comment]
It is pretty easy to do either, do you have a preference @terracoda?

**Comment by terracoda** [comment]
Ok, well, if they 
- remove the wall and add it back, the orientation can go back
- if they add the green balloon, and then remove it again (without interacting with the either balloon), the orientation **could** go back.

Once they move a balloon, it has to be gone until the sim is reset.

Would that work?

**Comment by jessegreenberg** [comment]
@terracoda sure! That is how it should behave with the above commit.  Could you please review/verify on phettest?

**Comment by terracoda** [comment]
Sure :-)

**Comment by terracoda** [comment]
@jessegreenberg, it works beautifully!

**Comment by jessegreenberg** [comment]
Great @terracoda, thank you! Closing this issue then.
